certified-connectors/OnePlan/apiDefinition.swagger.json

```diff
@@ -0,0 +1,553 @@
+                },
+                "List": {
+                  "type": "integer",
+                  "description": "List",
+                  "x-ms-visibility": "internal",
+                  "title": "",
+                  "default": 1
+                },
+                "Strategy": {
+                  "type": "string",
+                  "description": "Strategy",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "@{workflow()['tags']['flowDisplayName']}"
+                },
+                "Triggers": {
+                  "type": "array",
+                  "items": {
+                    "type": "integer",
+                    "format": "int32"
+                  },
+                  "x-ms-visibility": "internal",
+                  "default": [1]
+                }
+              },
+              "required": ["EventUrl", "List", "Strategy", "Triggers"]
+            }
+          }
+        ]
+      }
+    },
+    "/events/bylist/0": {
+      "x-ms-notification-content": {
+        "schema": {
+          "type": "object",
+          "properties": {
+            "Id": {
+              "type": "string",
+              "description": "Id"
+            },
+            "Name": {
+              "type": "string",
+              "description": "Name"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
+          }
+        },
+        "summary": "When a Plan Gets Updated",
+        "operationId": "PlanUpdateTrigger",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "EventUrl": {
+                  "type": "string",
+                  "description": "EventUrl",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
+                },
+                "Strategy": {
+                  "type": "string",
+                  "description": "Strategy",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "@{workflow()['tags']['flowDisplayName']}"
+                },
+                "Triggers": {
+                  "type": "array",
+                  "items": {
+                    "type": "integer",
+                    "format": "int32"
+                  },
+                  "x-ms-visibility": "internal",
+                  "default": [1]
+                }
+              },
+              "required": ["EventUrl", "Strategy", "Triggers"]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "DynamicResponseGetListSchema": {
+      "type": "object",
+      "x-ms-dynamic-schema": {
```

Comment:
Please fix the below Swagger validation error. 

##[error]The 'x-ms-dynamic-schema' extension property 'value-path' points to schema properties. You need to define the 'x-ms-dynamic-properties' extension which points to schema object. ValuePath: /definitions/DynamicResponseGetListSchema/x-ms-dynamic-schema/value-path

Please refer the below schema for reference. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/apiDefinition.swagger.schema.json#L2251